### ISSUE TITLE
Buggy behavior when using focal zoom and automatic containment

### ISSUE THREAD

**Issue by farzadmf** [timmywil/panzoom#353]
### Subject of the issue
When I use focal zoom and also set `contain` to `true`, the behavior is not exactly correct

### Your environment
* jquery.panzoom version: Latest
* jquery version: 3.2.1
* Chrome 60.0.3112.101

### Steps to reproduce
I use the example in the `demos` folder (the very last one that has focal zoom), and I set `contain = 'automatic'` in the options

### Expected behaviour
Zooming should happen under the mouse cursor and the element should be contained in its parent element. Sample expected behavior: I don't move the mouse and just zoom in (until I reach the maximum scale), and then zoom out (until I reach the minimum scale); I expect the element to be where it was before I started zooming

### Actual behaviour
When zooming, for the first two "zoom"'s, the behavior is fine (the zoom is done exactly under mouse cursor), but then, the element starts to move a little bit, causing the zoom pivot to not be exactly under mouse cursor. The situation is worse when I'm zooming out.
In summary, if I don't move the mouse and zoom in and zoom back out, the element will end up moving to the top or bottom!

PS: In fact, I noticed that the speed that you do the zoom (I'm doing it with the mouse wheel) affects the zoom-in/out behavior, and the behavior isn't consist
